src/backend/service/util.py

```diff
@@ -3,16 +3,25 @@
 """
 
 from enum import Enum
+from typing import Optional
 
 import boto3
+from aws_lambda_powertools.event_handler import Response
 from aws_lambda_powertools.logging import Logger
+from botocore.config import Config
 from botocore.exceptions import ClientError
 from cachetools.func import ttl_cache
 
 from .exceptions import ExternalServiceException, PermissionException
 
 logger = Logger()
 
+cors_headers = {
```

Comment:
The convention is typically to have constants be all caps so like:
```suggestion
CORS_HEADERS = {
```